src/AnyAddress.h

```diff
@@ -17,15 +18,16 @@
 #include "Cardano/AddressV3.h"
 #include "Cosmos/Address.h"
 #include "Decred/Address.h"
-#include "Ethereum/Address.h"
 #include "Elrond/Address.h"
+#include "Ethereum/Address.h"
 #include "Kusama/Address.h"
 #include "NEAR/Address.h"
 #include "NEO/Address.h"
 #include "Nano/Address.h"
 #include "Polkadot/Address.h"
 #include "Ronin/Address.h"
 #include "Solana/Address.h"
+#include "TrustWalletCore/TWBlockchain.h"
```

Comment:
```suggestion
#include <TrustWalletCore/TWBlockchain.h>
```

Reply:
This is added automatically, let me fix it